server/database.ts

```diff
@@ -29,6 +29,11 @@ export function initDB(dbFile: string): Database {
             Timestamp INTEGER, 
             FOREIGN KEY(Username) REFERENCES Users(Username)
         );`);
+
+        newDB.run(`CREATE TABLE if not exists "PostDislikes" (
+            ID TEXT PRIMARY KEY, 
+            Username TEXT
+        );`);
```

Comment:
Per `server/api/react.ts`, add a boolean `Disliked` value here instead of creating and destroying objects whenever a dislike happens.